src/senaite/core/api/dtime.py

```diff
@@ -436,3 +437,39 @@ def to_localized_time(dt, long_format=None, time_only=None,
                 formatstring = "[INTERNAL ERROR]"
         time_str = date_to_string(dt, formatstring, default=default)
     return time_str
+
+
+def get_relative_delta(from_dtime, to_dtime=None):
+    """Returns the relative delta between two datetimes. If to_dtime is None,
+    compares current datetime with from_dtime
+    """
```

Comment:
Would you mind to provide the docstrings in API functions in the same way we did for the `to_localized_time`, e.g.:

```python
def get_relative_delta(from_dtime, to_dtime=None):
    """Calculate the relative delta between two dates
    
    If `to_dtime` is None, the current datetime is used.

    :param from_dtime:
    :type from_dtime: str/datetime/DateTime
    :param to_dtime:  
    :type to_dtime: str/datetime/DateTime
    :returns: relativedelta object, e.g. `relativedelta(hours=+3)`
    :rtype: object
    """

Reply:
Fixed with 0aed61205f5eadae5d685434ceae79561ef32638